docs/apache-airflow/migrations-ref.rst

```diff
@@ -39,7 +39,10 @@ Here's the list of all the Database Migrations that are executed via when you ru
 +---------------------------------+-------------------+-------------------+--------------------------------------------------------------+
 | Revision ID                     | Revises ID        | Airflow Version   | Description                                                  |
 +=================================+===================+===================+==============================================================+
-| ``677fdbb7fc54`` (head)         | ``1949afb29106``  | ``2.10.0``        | add new executor field to db                                 |
+| ``bff083ad727d`` (head)         | ``677fdbb7fc54``  | ``2.9.1``         | Remove ``idx_last_scheduling_decision`` index on             |
+|                                 |                   |                   | last_scheduling_decision in dag_run table                    |
++---------------------------------+-------------------+-------------------+--------------------------------------------------------------+
+| ``677fdbb7fc54``                | ``1949afb29106``  | ``2.10.0``        | add new executor field to db                                 |
```

Comment:
My first migration PR here. I am wondering if we need to do some work around for the head here. Meaning head is pointing to 2.9.1, but we already added an index before for 2.10. If someone knows we need to handle this differently would be helpful.

Reply:
okay, tried doing so in https://github.com/apache/airflow/pull/39275/commits/bcb8a9dfc4e247812164ba0ef9ee53a6c797dcf0, please check if that's the right way.

I tested upgrading and downgrading from those airflow versions/db migration revisions and seems to work fine.

Reply:
LGTM.